homeassistant/components/sensor/swiss_public_transport.py

```diff
@@ -111,7 +112,7 @@ def device_state_attributes(self):
             ATTR_DEPARTURE_TIME2: self._opendata.connections[2]['departure'],
             ATTR_START: self._opendata.from_name,
             ATTR_TARGET: self._opendata.to_name,
-            ATTR_REMAINING_TIME: '{}'.format(remaining_time),
+            ATTR_REMAINING_TIME: '{}'.format(self._remaining_time),
```

Comment:
We don't want to store relative (to now) times in the state machine as they spam the it.

Reply:
I see, it is an existing attribute. I don't use it / dont see a need for it - don't know what is the adoption.
Agree this does not necessarily be in state machine and can be calculated in a template only by whom who needs it.
Do you recommend deprecating this attribute?

Reply:
I would say so yes. @fabaff do you agree?